c9/user_conf.py

```
@@ -0,0 +1,57 @@
+
+import sys
+import os
+from logger_config import setup_logging as _setup_logging  # type: ignore
+
+from dotenv import load_dotenv
+try:
+    load_dotenv()
+except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)